adapters/grid/grid.go

```diff
@@ -282,6 +300,35 @@ func setImpExtData(imp openrtb2.Imp) openrtb2.Imp {
 	return imp
 }
 
+func fixNative(req json.RawMessage) (json.RawMessage, error) {
+	var gridReq map[string]interface{}
+	var parsedRequest map[string]interface{}
+
+	if err := json.Unmarshal(req, &gridReq); err != nil {
```

Comment:
Right before executing this function you `reqJSON, err := json.Marshal(request)`. Do you think it will be easier to pass `openrtb2.BidRequest` here instead of `req json.RawMessage`?

Reply:
I guess it would be easier to operate on abstract json since we need to change its structure

Reply:
I see it now, ok!